### PR TITLE
Update Windows instructions

### LINKED ISSUE #15155: Windows points to old Ubuntu instructions
## Steps to Reproduce
Head on over to https://mooseframework.inl.gov/getting_started/installation/windows10.html. Following those instructions will force you back to the old Ubuntu instructions.

## Impact
missing links, use of out-dated moose-environment packages

### PR DESCRIPTION
**Pull request by milljm**
Point the Linux part of our WSL instructions to the Conda Install instructions.

Closes #15155

### CODE REVIEW

**Review by cticenhour — DISMISSED** [review]

**Review by cticenhour — APPROVED** [review]
Will merge when green (unless you want this in immediately since it's just a doco change....your call).

**Comment by milljm** [comment]
@cticenhour thanks! I'll just merge manually when I have a moment... which is now! lol

**Pull request merged**
